modules/storage-api/src/main/java/org/apache/ignite/internal/storage/index/StorageIndexDescriptor.java

```diff
@@ -26,40 +26,29 @@
 import org.apache.ignite.internal.type.NativeType;
 import org.apache.ignite.internal.type.NativeTypes;
 
-/**
- * Index descriptor.
- */
+/** Index descriptor. */
```

Comment:
Please revert all these useless javadoc updates

Reply:
I didn't see anywhere that this was prohibited.

Reply:
> Should I start a discussion on dev-list for you?
Only if you yourself are interested in it.

Reply:
This task is very big, I don’t want to waste my mental energy on such a large non-functional change.

I have slightly revised my approach and will make such changes only where I actually change the code and around it only if this one needs to be changed according to the meaning of the task.